mitodl/open-discussions
# Added login/register UI

#### What are the relevant tickets?
Fixes #700 
Fixes #701 
Fixes #676 

#### What's this PR do?
This adds UI for login and registration via email

#### How should this be manually tested?

- Go to /register and confirm you can register (receive confirmation email, fill in details, etc)
- Go to /login and verify you can login
- Go to /register and try to register with the user you created, confirm you are redirected to enter a password instead
- Go to /login and enter a nonexistent email, confirm you are prompted to create an account and clicking the button sends the email and allows you to complete that flow.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
